cs-utilities/src/test/java/io/cloudslang/content/utilities/actions/DefaultIfEmptyTest.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.utilities.actions;
+
+import org.junit.Test;
+
+import java.util.Map;
+
+import static io.cloudslang.content.constants.OutputNames.RETURN_CODE;
+import static io.cloudslang.content.constants.OutputNames.RETURN_RESULT;
+import static junit.framework.Assert.assertEquals;
+
+/**
+ * Created by moldovai on 8/22/2017.
+ */
+public class DefaultIfEmptyTest {
+
+    private final DefaultIfEmptyAction d = new DefaultIfEmptyAction();
+
+    @Test
+    public void testWithBlankString() {
+        final Map<String, String> result = d.execute("   ", "string", "  ");
+        assertEquals("0", result.get(RETURN_CODE));
+        assertEquals("string", result.get(RETURN_RESULT));
+    }
+
+    @Test
+    public void testWithEmptyString() {
+        final Map<String, String> result = d.execute("", "string", "");
+        assertEquals("0", result.get(RETURN_CODE));
```

Comment:
use ReturnCodes.SUCCESS/ ReturnCodes.FAILURE instread of "0"/"-1" from cs-commons